draft-packages/menu/KaizenDraft/Menu/StatelessMenu.tsx

```diff
@@ -1,31 +1,111 @@
-import { default as React, ReactElement, useRef } from "react"
-
-import { GenericMenuProps, render } from "./Menu"
+import { default as React, useEffect, useState } from "react"
+import ReactDOM from "react-dom"
 import styles from "./styles.scss"
+import MenuDropdown from "./MenuDropdown"
 
 export type StatelessMenuProps = {
+  /**
+   * Whether the menu is to be used on the left or right
+   * side of the viewport. If left, the left of the dropdown
+   * is aligned to the left of the button (and vice versa)
+   * @default "left"
+   */
+  align?: "left" | "right"
+
+  /**
+   * The width of the dropdown.
+   * "default": a fixed width of 248px
+   * "contain": contain the children's width (will be same width as children)
+   * @default "default"
+   */
+  dropdownWidth?: "default" | "contain"
+
+  automationId?: string
+  dropdownId?: string
+  /**
+   * Determines when the menu should automatically hide.
+   * @default: "on"
+   */
+  autoHide?: "on" | "outside-click-only" | "off"
+  /**
+   * The content to appear inside the dropdown when it is open
+   */
+  children: React.ReactNode
+  portalSelector?: string
```

Comment:
Could we please add a JSDoc comment here explaining what a `portalSelector` is? It might be confusing to people unfamiliar with it otherwise.